__tests__/app.test.js

```diff
@@ -33,50 +33,68 @@ describe('GET /api/topics', () => {
     })
 })
 
-describe('GET /api/articles/:article_id', () => {
-    it('returns an article object with the correct properties', () => {
-        return request(app)
-            .get(`/api/articles/4`)
-            .expect(200)
-            .then(({
-                body
-            }) => {
-                expect(body.article).toMatchObject({
-                    article_id: 4,
-                    title: "Student SUES Mitch!",
-                    topic: "mitch",
-                    author: "rogersop",
-                    body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
-                    created_at: expect.any(String),
-                    votes: 0,
-                })
-            })
-    })
-    it('should return a 400 with a bad request if the id is invalid', () => {
-        return request(app)
-            .get(`/api/articles/bad`)
-            .expect(400)
-            .then(({
-                body
-            }) => {
-                expect(body).toEqual({
-                    msg: 'bad request'
-                })
-            })
-    })
-    it('should return a 404 not found when given a valid number id that does not exist', () => {
-        return request(app)
-            .get(`/api/articles/153`)
-            .expect(404)
-            .then(({
-                body
-            }) => {
-                expect(body).toEqual({
-                    msg: 'article ID not found'
-                })
-            })
-    })
-})
+// describe('GET /api/articles/:article_id', () => {
```

Comment:
Ideally these shouldn't even exist on this branch, as this branch is for a different feature. 

I'd make sure you're branching from main each time, then all the branches are independent, and all feed into main